src/libraries/EndpointStructs.sol

```diff
@@ -71,11 +72,12 @@ library EndpointStructs {
     ///      The wire format is as follows:
     ///      - NTT_PREFIX - 4 bytes
     ///      - amount - 8 bytes
+    ///      - numDecimals - 1 byte
```

Comment:
Can you update the docstring to have the wire format in the correct order pls?

Reply:
it is in the right order.
```struct NormalizedAmount {
    uint64 amount;
    uint8 decimals;
}
```

Reply:
Not in the encoded payload it isn't: https://github.com/wormhole-foundation/example-native-token-transfers/blob/a614dd8213fdb47967945f6b7d62e9ee83e2a4c3/src/libraries/EndpointStructs.sol#L96-L102. One or the other should be flipped around I think